client/src/lit/interactive-example/with-choices.js

```diff
@@ -102,7 +102,7 @@ export const InteractiveExampleWithChoices = (Base) =>
           </header>
           <div
             class="choice-wrapper"
-            @click=${this.#choiceClick}
+            @focus=${this.#choiceSelect}
```

Comment:
Need to verify that this doesn't regress on Safari, which doesn't (always?) focus implicitly on click.

Reply:
Deployed to https://select-ix-choice-on-focus.review.mdn.allizom.net/en-US/docs/Web/CSS/text-align#try_it and verified in Safari that it works, both via mouse click and keyboard tabbing.